compiler/circle-mpqsolver/src/core/TestHelper.test.cpp

```diff
@@ -208,5 +208,36 @@ bool isFileExists(const std::string &path)
 }
 
 } // namespace io_utils
+
+namespace data_utils
+{
+
+class SampleDataInput final : public core::DataProvider
```

Comment:
optional) not understanding much about this but how about `SingleDataProvider` ?

Reply:
> optional) not understanding much about this but how about `SingleDataProvider` ?

Yes. Looks better. Thank you.